src/main/java/gov/acwi/wqp/etl/activity/ActivityProcessor.java

```diff
@@ -0,0 +1,171 @@
+package gov.acwi.wqp.etl.activity;
+
+import gov.acwi.wqp.etl.Application;
+import gov.acwi.wqp.etl.ConfigurationService;
+import gov.acwi.wqp.etl.biodata.activity.BiodataActivity;
+import java.time.LocalDateTime;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ActivityProcessor implements ItemProcessor<BiodataActivity, Activity>{
+
+	static final String DEFAULT_SAMPLE_MEDIA = "Biological";
```

Comment:
I'm a little surprised these weren't flagged for not having a public/protected/private modifier...

Reply:
I was too in that I didn't know you could do this. Without the access modifier it has `default` access, also known as `package-private`, which limits access to this constant to classes within its package.  I added it based on feedback from an IntelliJ code inspection, but I can add explicit access modifiers if you prefer as well.